### ISSUE TITLE
Lua error after recent tbc update

### ISSUE THREAD

**Issue by Blumberg1337** [sandervspl/BuffTimers#2]
2x FrameXML\InterfaceOptionsFrame.lua:625: bad argument #1 to 'strlower' (string expected, got boolean)
[string "=[C]"]: ?
[string "@FrameXML\InterfaceOptionsFrame.lua"]:625: in function `InterfaceOptions_AddCategory'
[string "@BuffTimers\Options-Options.lua"]:47: in main chunk

Locals:
(*temporary) = true
(*temporary) = "string expected, got boolean"

**Comment by boktai1000** [comment]
Posted it as well here: https://www.curseforge.com/wow/addons/bufftimers?comment=31

```
Message: Interface\FrameXML\InterfaceOptionsFrame.lua:625: bad argument #1 to 'strlower' (string expected, got boolean)
Time: Mon May 23 14:49:05 2022
Count: 1
Stack: Interface\FrameXML\InterfaceOptionsFrame.lua:625: bad argument #1 to 'strlower' (string expected, got boolean)
[string "=[C]"]: ?
[string "@Interface\FrameXML\InterfaceOptionsFrame.lua"]:625: in function `InterfaceOptions_AddCategory'
[string "@Interface\AddOns\BuffTimers\Options.lua"]:47: in main chunk

Locals: (*temporary) = true
(*temporary) = "string expected, got boolean"
```

 

 

```
Message: (null)
Time: Mon May 23 14:49:11 2022
Count: 1
```

 

```
Message: (null)
Time: Mon May 23 14:49:11 2022
Count: 1
```

 
```

Message: (null)
Time: Mon May 23 14:49:11 2022
Count: 1
```

![image](https://user-images.githubusercontent.com/9054563/169952588-7c24a0d7-2c95-4d22-a8f5-555cf89b4ee8.png)

**Comment by Mathizsias** [comment]
Reverting back to a previous release solved this particular issue for me.

https://github.com/sandervspl/BuffTimers/releases/tag/v1.5.5

You can then set it to be ignored in WoWUP or the CurseForge client.

**Comment by Blumberg1337** [comment]
@Mathizsias Thank you for your help. I just wanted to report the bug here, as it seems to be a good place for it. ;)
Hope this helps improving the addon, because I like it. It got posted on curseforge, as well as @boktai1000 mentioned.

**Comment by sandervspl** [comment]
This issue was fixed in v1.5.7. Are you still experiencing this bug? @Blumberg1337

**Comment by Blumberg1337** [comment]
@sandervspl No, just forgot to close this issue. Thanks!
